### PR TITLE
Mentions feature added

### PR DESCRIPTION
**Pull request by mharshita**
fixes: #15 

Mentions feature added.

https://user-images.githubusercontent.com/60281688/129571706-776c7055-94a9-410a-8f57-fd5a787c3ab0.mp4

### CODE REVIEW

File: src/assets/scss/components/comments.scss [hunk 1]
```diff
@@ -30,7 +32,64 @@
...
+  border: 1px solid silver;
+  font-family: $font-family;
+  font-size: $font-size-general;
+}
+
+.mentions--multiLine .mentions__highlighter {
+  padding: 9px;
+}
+
+.mentions--multiLine .mentions__input {
```
Comment (kbhutani0001) [inline src/assets/scss/components/comments.scss:71]: nested styles please

File: src/components/Comments.js [hunk 2]
```diff
@@ -76,7 +79,25 @@ const Comments = (props) => {
...
+              id
+              display: username
+            }
+          }`
+        },
+        {
+          withCredentials: true
+        }
+      )
+      setUsers(response.data.data.users)
```
Comment (kbhutani0001) [inline src/components/Comments.js:97]: null checks here please. if the request fails, this will break the app.

**Comment by kbhutani0001** [comment]
Why is this still draft @mharshita ?

**Comment by mharshita** [comment]
@kbhutani0001  actually currently it is working like this : 

https://user-images.githubusercontent.com/60281688/128857318-9878b99f-6c94-4fa3-a601-2d04d39735c5.mp4

**Comment by mharshita** [comment]
Through this PR we would be able to add mentions but mentioned usernames would not be converted to links.

File: .github/workflows/github-ci.yml [hunk 3]
```diff
@@ -1,9 +1,35 @@
 name: Run Tests
 
+<<<<<<< HEAD
```
Comment (cyntss) [inline .github/workflows/github-ci.yml:3]: we need to remove these lines that come from a merge conflict

File: .github/workflows/github-ci.yml [hunk 4]
```diff
@@ -1,9 +1,35 @@
...
+          path: node_modules
+          key: node_modules-${{ hashFiles('package-lock.json') }}
+
+      - name: Install dependencies
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: npm install
+
+  test_lints_sass:
+    needs: cache_node_modules
+=======
```
Comment (cyntss) [inline .github/workflows/github-ci.yml:27]: this one too

File: .github/workflows/github-ci.yml [hunk 5]
```diff
@@ -1,9 +1,35 @@
...
 on: [pull_request]
 
 jobs:
   test_lints:
+>>>>>>> main
```
Comment (cyntss) [inline .github/workflows/github-ci.yml:32]: and this one

**Review by cyntss — CHANGES_REQUESTED** [review]

**Comment by mharshita** [comment]
Done! thanks @cyntss :)

**Review by cyntss — APPROVED** [review]

**Pull request merged**